test/run-k8s-integration.sh

```diff
@@ -31,10 +31,12 @@ readonly parallel_run=${PARALLEL:-}
 
 make -C "${PKGDIR}" test-k8s-integration
 
-go install sigs.k8s.io/kubetest2@latest;
-go install sigs.k8s.io/kubetest2/kubetest2-gce@latest;
-go install sigs.k8s.io/kubetest2/kubetest2-gke@latest;
-go install sigs.k8s.io/kubetest2/kubetest2-tester-ginkgo@latest;
+# Choose an older Kubetest2 commit version instead of using @latest 
+# because of a regression in https://github.com/kubernetes-sigs/kubetest2/pull/183.
```

Comment:
please add a comment about what should we do when we need to update this

Reply:
Let's wait for engprod to reach a final solution.

Reply:
"contact engprod oncall and ask about what is the version they are using for internal jobs" is a good place to start. let's merge with that instruction first

Reply:
Done, comment added.